packages/frontend/src/state/controller/hooks.ts

```diff
@@ -210,6 +210,17 @@ export const useGetTwapEthPrice = () => {
   return getTwapEthPrice
 }
 
+export const useGetTwapSqueethPrice = () => {
```

Comment:
maybe + useGetSpotPrice using pool tick

Reply:
Yep, using this in the new approach [here](https://github.com/opynfinance/squeeth-monorepo/pull/521)!